react-facet/docs/game-ui-development/shared-facet.md

```diff
@@ -46,7 +52,28 @@ const userNameFacet = sharedSelector(({ user }) => user.name, [profileFacet])
 
 The user name facet will only hold the user name as data and can be consumed like this:
 
-```tsx
+```tsx twoslash
+// @esModuleInterop
+// @errors: 2322
+import { render } from '@react-facet/dom-fiber'
+import { sharedFacet, sharedSelector } from '@react-facet/shared-facet'
+
+interface UserFacet {
+  user: {
+    name: string
+    lastname: string
+  }
+}
+
+const profileFacet = sharedFacet<UserFacet>('data.user', {
+  user: {
+    name: 'Jane',
+    lastname: 'Doe',
+  },
+})
+
+const userNameFacet = sharedSelector(({ user }) => user.name, [profileFacet])
+// ---cut---
 const UserData = () => {
   // This will print Jane
   return (
```

Comment:
`<fast-text>` doesn't seem to accept `SharedFacet`. https://jolly-stone-0d2f00e03-58.westeurope.azurestaticapps.net/docs/game-ui-development/shared-facet#dynamic-selectors